packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -169,13 +169,12 @@ export function loadBasedir(basedir, options = {}) {
  * @param {string} referrer  URL of file or directory containing the config file
  * @param {string} specPath  Path found in a `sourceSpec` or `bundleSpec` property
  *
- * @returns {Promise<string>} the absolute path corresponding to `specPath` if it can be
+ * @returns {string} the absolute path corresponding to `specPath` if it can be
```

Comment:
This is a case where the path is dynamic - not determined until runtime. Let's keep it async, please.

Using sync I/O for a sort of linker/loader makes sense, but otherwise, let's not force clients into the world of sync I/O.